pytests/test_event_time.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime, timedelta
+
+from bytewax.dataflow import Dataflow
+from bytewax.execution import run_main
+from bytewax.inputs import ManualInputConfig
+from bytewax.window import TumblingWindowConfig, EventClockConfig, TestingClockConfig
+from bytewax.outputs import TestingOutputConfig
+
+
+def test_event_time_processing():
+    """
+    Test used to validate the EventClockConfig workings.
+    """
+    start_at = datetime.now()
+    late = timedelta(seconds=10)
+    window_length = timedelta(seconds=5)
+
+    # Utility functions to generate "temp" events
+    def temp(val, seconds):
+        time = start_at + timedelta(seconds=seconds)
+        return None, {"type": "temp", "time": time, "value": val}
+
+    # Utility functions to generate wait events (used for advancing the clock)
+    def wait():
+        return None, {"type": "temp", "time": start_at}
+
+    def input_builder(worker_index, worker_count, state):
+        # Each yield advances the clock by 1 second
+        # This should be processed in the first window
+        yield temp(1, 1)
```

Comment:
Appreciate the comments! Optionally, you could modify the events to be strings instead of temp values, so their values could be more revealing, i.e. "window 1", "drop since late", "window 2".